internal/operators/cnv/manifest.go

```diff
@@ -94,6 +103,13 @@ func hco(config manifestConfig) ([]byte, error) {
 	return executeTemplate(data, "cnvHCO", cnvHCOManifestTemplate)
 }
 
+func hpp(diskThresholdGi int64) ([]byte, error) {
```

Comment:
Please use more meaningful name

Reply:
This is the shorthand for hostpath provisioner, similarly to how we shorthand cnv/hco:
https://github.com/openshift/assisted-service/pull/3131/files#diff-448ecdc1e6c9432844dd5dd2c0fe9ed770432db8e41ca79ef62806c6fe677ad0R99